### PR TITLE
refactor(kalman_filter): prefix package and namespace with autoware

### PR DESCRIPTION
**Pull request by esteve**
## Description

prefix package and namespace with autoware

## Related links

**Parent Issue:**

- Link https://github.com/autowarefoundation/autoware/issues/4569

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: common/autoware_kalman_filter/include/autoware_kalman_filter/kalman_filter.hpp
Comment (SakodaShintaro) [inline common/autoware_kalman_filter/include/autoware_kalman_filter/kalman_filter.hpp:1]: As well as the comments on another pull request, is this structure of the include directory okay?
https://github.com/autowarefoundation/autoware.universe/pull/7790#pullrequestreview-2155145037

**Review by technolojin — COMMENTED** [review]
Except the include directory topic, LGTM

**Comment by esteve** [comment]
Back to draft to fix the issue in https://github.com/autowarefoundation/autoware.universe/pull/7787#discussion_r1663650652

**Comment by esteve** [comment]
@SakodaShintaro @technolojin fixed in 8b607239ca1d2975aea0cd3b1e91ae4f2ed7676f

**Review by SakodaShintaro — APPROVED** [review]
I have confirmed that logging_simulator works well.
LGTM(for /localization)

**Review by technolojin — APPROVED** [review]
In terms of Perception, LGTM

**Comment by Motsu-san** [comment]
@manato @YoshiRi Could you review this pull-request?

**Review by YoshiRi — APPROVED** [review]
logging simulator worked in my local environment. LGTM.

**Comment by Motsu-san** [comment]
@esteve Could you resolve the conflicts?

**Comment by esteve** [comment]
@Motsu-san fixed

**Comment by YoshiRi** [comment]
I think we need approve from each components maintainers.

- Sensing: @manato 
- Planning/Control: @TakaHoribe 
- Architect: @yukkysaito 

Hi, could you approve this PR? We have already checked this PR.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -22,8 +23,7 @@ common/global_parameter_loader/** ryohsuke.mitsudome@tier4.jp
 common/glog_component/** takamasa.horibe@tier4.jp
 common/goal_distance_calculator/** taiki.tanaka@tier4.jp
 common/interpolation/** fumiya.watanabe@tier4.jp takayuki.murooka@tier4.jp
-common/kalman_filter/** koji.minoda@tier4.jp takeshi.ishita@tier4.jp yukihiro.saito@tier4.jp
-common/object_recognition_utils/** shunsuke.miura@tier4.jp takayuki.murooka@tier4.jp yoshi.ri@tier4.jp
+common/object_recognition_utils/** satoshi.tanaka@tier4.jp shunsuke.miura@tier4.jp takayuki.murooka@tier4.jp yoshi.ri@tier4.jp
```
Comment (TakaHoribe) [inline .github/CODEOWNERS:26]: Unrelated change?

Comment (esteve) [reply .github/CODEOWNERS]: Fixed, must have been an error resolving the conflicts.

**Review by manato — APPROVED** [review]
Regarding `autoware_bytetrack`, LGTM. Thank you for your contribution!

**Comment by vividf** [comment]
@kminoda @yukkysaito 
It seems that we need a code owner review for this PR.

**Review by yukkysaito — APPROVED** [review]
